src/mon/OSDMonitor.cc

```diff
@@ -6008,6 +6008,12 @@ int OSDMonitor::prepare_command_pool_set(map<string,cmd_vartype> &cmdmap,
 	return -EEXIST;
       return 0;
     }
+    if (n > (unsigned)g_conf->mon_max_pool_pg_num) {
+      ss << "'pg_num' must be greater than 0 and less than or equal to "
+      << g_conf->mon_max_pool_pg_num
```

Comment:
Can you make sure these lines are `<< `aligned too? 

Also please squash these three commits into a single one.
  https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst might be help.